docs/source/deploypeer/peerplan.md

```diff
@@ -0,0 +1,94 @@
+# Planning for a production peer
+
+Audience: Architects, network operators, users setting up a production Fabric network who are familiar with Transport Layer Security (TLS), Public Key Infrastructure (PKI) and Membership Service Providers (MSPs).
+
+Peer nodes are a fundamental element of a Fabric network because they host ledgers and smart contracts that are used to encapsulate the shared processes and shared information in a blockchain network. These instructions assume you are already familiar with the concept of a [peer](https://hyperledger-fabric.readthedocs.io/en/{BRANCH}/peers/peers.html) and provides guidance for the various decisions you will have to make about a peer you will deploy and join to a production Fabric network channel. If you need to quickly stand up a network for education or testing purposes, check out the [Fabric test network](../test_network.html).
```

Comment:
[peer](https://hyperledger-fabric.readthedocs.io/en/{BRANCH}/peers/peers.html)

Since this links to a local document we don't need to use the {BRANCH} link here... We should just use
[peer](../peers/peers.html) or whatever the right path is based on where this document resided in the toc.